features/product/managing_product_attributes/adding_select_product_attribute.feature

```diff
@@ -10,21 +10,21 @@ Feature: Adding a new select product attribute
 
     @ui @javascript
     Scenario: Adding a new select product attribute
-        Given I want to create a new select product attribute
-        When I specify its code as "mug_material"
+        When I want to create a new select product attribute
+        And I specify its code as "mug_material"
         And I name it "Mug material" in "English (United States)"
-        And I add material "-100% Banana Skin"
+        And I add value "-100% Banana Skin"
```

Comment:
Value is kinda misleading, my first - AttributeValue, the second - the same. What about naming it an "option" or a "select option"?

Reply:
But we have values in configuration of a product attribute, so I wanted to be consistent with the layout:
![zrzut ekranu 2017-09-28 o 15 54 29](https://user-images.githubusercontent.com/6140884/30970417-5ac0fc5e-a465-11e7-8cf1-a429068036bb.png)

Reply:
I see the reasoning behind it, but it's still confusing, not sure whether we should solve it already or leave it as is :)